src/Platform/Microsoft.Testing.Platform/CommandLine/OptionRecord.cs

```diff
@@ -3,7 +3,8 @@
 
 namespace Microsoft.Testing.Platform.CommandLine;
 
-internal sealed class OptionRecord(string option, string[] arguments)
+[Experimental("TPEXP", UrlFormat = "https://aka.ms/testingplatform/diagnostics#{0}")]
+public sealed class OptionRecord(string option, string[] arguments)
```

Comment:
Double-checking if the naming is intended. Do we want to rename `option` to `name`? Or are we fine with having "option" duplicated in type name and the property/parameter names?

Reply:
Not sure also if `CommandLineOption` is a better name than `OptionRecord`